PromiseKit.xcodeproj/project.pbxproj

```diff
@@ -1071,7 +1071,7 @@
 				LD_RUNPATH_SEARCH_PATHS = "$(inherited) @executable_path/Frameworks @loader_path/Frameworks";
 				PRODUCT_BUNDLE_PACKAGE_TYPE = FMWK;
 				SKIP_INSTALL = YES;
-				SWIFT_TREAT_WARNINGS_AS_ERRORS = NO;
+				SWIFT_TREAT_WARNINGS_AS_ERRORS = YES;
```

Comment:
As you can see at https://github.com/mxcl/PromiseKit/pull/1144#issuecomment-637866172 you should keep `SWIFT_TREAT_WARNINGS_AS_ERRORS = NO;`.